.husky/pre-commit

```diff
@@ -1,4 +1 @@
-#!/bin/sh
```

Comment:
received a deprecation warning about this so removed (has also been removed from the template project)